lib/ret_web/controllers/api-internal/v1/rewrite_assets_controller.ex

```diff
@@ -0,0 +1,24 @@
+defmodule RetWeb.ApiInternal.V1.RewriteAssetsController do
+  use RetWeb, :controller
+
+  alias Ret.{RoomObject, Project, Scene}
+
+  def post(conn, %{"old_domain" => old_domain, "new_domain" => new_domain})
+      when is_binary(old_domain) and is_binary(new_domain) and old_domain != "" and new_domain != "" do
+    conn = put_resp_header(conn, "content-type", "application/json")
+
+    old_domain_url = "https://#{old_domain}"
```

Comment:
nit: since it's coming from our servers this is fine, but to be safe we could add a regex check that ensures neither old_domain or new_domain has "https://" or "http://" in the value